resources/qml/delegates/PlayableMediaMessage.qml

```diff
@@ -287,13 +292,72 @@ Column {
                     height: videoOutput.contentRect.height
                     propagateComposedEvents: true
                 }
-                anchors.fill: parent
-                fillMode: VideoOutput.PreserveAspectFit
-                source: media
             }
         }
     }
+    // Audio player
+    // TODO: share code with the video player
+    Rectangle {
```

Comment:
Please put this in its own file, if they share nothing currently and change MessageDelegate accordingly.